msteams-platform/concepts/deploy-and-publish/overview.md

```diff
@@ -1,56 +1,72 @@
 ---
 title: Distribute your app
-description: Describes the three options for distributing your app`
+description: Describes the three options for distributing your app
```

Comment:
```suggestion
description: Describes the options for distributing your Microsoft Teams app.
```
Keeps number of options generic in case it changes. Also added words to help with SEO.